### PR TITLE
Eseries integration

### PR DESCRIPTION
**Pull request by mechterp10**
This is a first pass to get E-series urdfs in order. I need to do some additional testing and determine how I want to properly construct these with our Stations.

Additionally, the .stl in this repository will need to be replaced by .dae files generated by Kevin. The current models are horrible.

### CODE REVIEW

File: ur_description/urdf/ur10e.urdf.xacro [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+    </inertial>
+  </xacro:macro>
+
+
+  <xacro:macro name="ur10_robot" params="prefix joint_limited">
+
+    <link name="${prefix}base_link" >
+      <visual>
+        <geometry>
+          <mesh filename="package://ur_description/meshes/ur10/visual/base.dae"/>
```
Comment (adamrr) [inline ur_description/urdf/ur10e.urdf.xacro:74]: Should these be using ur10e instead?

File: ur_description/urdf/ur3e.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+    </inertial>
+  </xacro:macro>
+
+
+  <xacro:macro name="ur3_robot" params="prefix joint_limited">
+
+    <link name="${prefix}base_link" >
+      <visual>
+        <geometry>
+          <mesh filename="package://ur_description/meshes/ur3/visual/base.dae"/>
```
Comment (adamrr) [inline ur_description/urdf/ur3e.urdf.xacro:69]: ur3e ?

**Review by adamrr — APPROVED** [review]
Are these using the non-eseries models for now until Kevin re-creates meshes for eseries?

**Comment by mechterp10** [comment]
You are correct. I am going to have to make adjusts. Ill close the PR

**Review by joeporembski — APPROVED** [review]
Not sure if you need to close the PR or if the xarco files can be modified/appropriate files added.

The size of this commit is what worries me, and perhaps this is just more evidence we need to figure out the asset pipeline issue once we get through 2.1.

**Comment by mechterp10** [comment]
Closing this since we need to make some changes

**Pull request closed without merging**